### ISSUE TITLE
Update all 404 templates to the latest tsParticles version

### ISSUE THREAD

**Issue by matteobruni** [tsparticles/404-templates#50]
Many 404 templates are using old versions of [tsParticles](https://github.com/matteobruni/tsparticles).

I'd like to have all of them updated to the latest version.

**Comment by matteobruni** [comment]
> Could you please assign this issue to me? I updated all the templates to the latest version (2.3.4). Is it correct?

Yes it’s correct

**Comment by demisquare** [comment]
After I updated the versions, It didn't recognise the `tsparticles.load` method anymore.
First, I included this and I already checked if the path works.

`<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/tsparticles@2.3.4/tsparticles.min.js"></script>`

![image](https://user-images.githubusercontent.com/48382390/194867626-58ab7d89-73c3-4d98-876b-e60a7efcde6c.png)

Then, I changed the <script> tag to `<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/tsparticles@2.3.4/tsparticles.bundle.min.js"></script>` and every template worked but Blue Dots, Polygon (it shows less polygons than previous version) and Snowfall.

Here some images:
![image](https://user-images.githubusercontent.com/48382390/194872592-cdc05e87-ab69-4136-befc-1835ca2870fb.png)

![image](https://user-images.githubusercontent.com/48382390/194872647-c2ce9e49-7692-4aa3-b672-a6de4eeb09ca.png)

![image](https://user-images.githubusercontent.com/48382390/194872734-ea1da8b1-a701-4be5-a064-5f528e4c9b0d.png)

Previous version:

![image](https://user-images.githubusercontent.com/48382390/194872917-bef21b0b-f1ae-4440-9c13-e69317c9c1d1.png)

**Comment by matteobruni** [comment]
For the last one, maybe it's because there's the `inlineArrangement` property set (under `polygon`) that is no more supported, it has to be changed to this: `inline: { arrangement: old_value }`

For the other ones, I don't know what the issue is.

**Comment by demisquare** [comment]
I managed do fix Polygon :D for Snowfall, I noticed there's a broken link in app.js:23...

```
      image: {
        src: "http://www.dynamicdigital.us/wp-content/uploads/2013/02/starburst_white_300_drop_2.png",
        width: 10,
        height: 10
      }
```

Still, I can't understand how to fix Blue Dots. Maybe there's some unsupported fields in the js, I suppose. Can I send you the PR anyway, so you can check the problem?

**Comment by demisquare** [comment]
Here #55

**Comment by matteobruni** [comment]
Found the issue, there's a bug with `rgb(r, g, b)` strings, replate the `rgb(0,0,255)` with `#0000ff` and it will work

**Comment by matteobruni** [comment]
Solved by #55
